src/components/header/header.styles.js

```diff
@@ -2,26 +2,30 @@ import styled from 'styled-components';
 import { NavLink } from 'react-router-dom';
 
 export const NavContainer = styled.nav`
-  height: 100px;
-  margin-left: 80px;
-  margin-right: 80px;
   display: flex;
   flex-direction: row;
   justify-content: space-between;
   align-items: center;
+  margin-bottom: 55px;
 `;
 
 export const LogoContainer = styled.div`
   display: block;
+  margin-left: 80px;
+  margin-top: 33px;
```

Comment:
This is to vertically align the logo? The links wrapper below does the same by using `margin-top: 30px`, right?

Ann alternative is to set a fixed height of 100px to the `NavContainer` and let `display: flex` and `align-items: center` take care of centering the elements vertically.